sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -155,6 +157,12 @@ export class LinkEntity {
    * @returns {void}
    */
   protected async _ensureTokenRenewal(): Promise<void> {
+    // Clear the existing token renewal timer in case it hasn't already fired.
```

Comment:
Shouldnt this be "in case it has already been fired" ?

Reply:
I should probably just rephrase this entirely since it's not clear what 'fired' actually means :)

Reply:
Changing this to:
```suggestion
    // Clear the existing token renewal timer in case the previous timeout hasn't been reached.
```
Just waiting for the live test run to finish since it's close and I don't want to have to restart it!

Reply:
Well...  The `if` check doesnt really check whether the previous timeout has reached or not right?
It only checks whether the timer has been set or not. So, this is still misleading..

I would recommend going with just

```suggestion
    // Clear the existing token renewal timer.
```